common-model/src/main/java/com/sequenceiq/common/api/type/LoadBalancerType.java

```diff
@@ -0,0 +1,6 @@
+package com.sequenceiq.common.api.type;
+
+public enum LoadBalancerType {
+    PUBLIC,
+    PRIVATE
```

Comment:
The purpose of this module is to provide a cloud provider agnostic modeling of the cloud resources. So I don't feel any particular need to follow the cloud provide convention super closely. Especially since I only know the AWS convention, so following that might just mean we're contradicting the Azure convention.

I am considering changing these to INTERNAL and EXTERNAL, but I'm going to think about that and make the change later if necessary.